### DISCUSSION TITLE
Why does pressing Ctrl, Alt, Shift create a copy paste event?

### DISCUSSION THREAD

**Discussion by TobiasWallner** [jupyter-xeus/cpp-terminal#278] [Q&A]
I was just trying to make my application so, that it would not trigger a re-render on held down control keys when I noticed that they do not create a key event that I could catch but a create an empty copy paste event. 
How am I supposted to distinguish them from other 'real' copy paste events?

**Comment by TobiasWallner** [comment]
I mean the copy paste event is empty, so I could check for an empty string but there is no function that lets me get a reference of the string to just check the `size()` in an `Term::Event`. So the only possibility for me is to construct a new `std::string` using a `static_cast` and risk a `malloc`. But I don't want to `malloc` when I don't need it. Which would not be needed if `Term::Event` was a `std::variant` - emphasis on #276 here

**Reply by flagarde** [reply]
It's a bug I think. I imagine you are on Windows right? Other OS doesn't trigger event only on meta keys press. I think for now to be consistent we should skip them too

**Reply by TobiasWallner** [reply]
Yes I am on windows

**Reply by TobiasWallner** [reply]
If they were just empty events that would be great.

**Reply by flagarde** [reply]
Will add a PR to skip them for now

**Comment by flagarde** [comment]
@TobiasWallner Please submit an issue to not forget that should be fixed
